__tests__/app.test.js

```diff
@@ -14,22 +14,24 @@ beforeEach(() => {
 
 describe("app testing", () => {
   describe("/api/topics", () => {
-    test("GET 200: returns with a status of 200 and an array", async () => {
-      await request(app).get("/api/topics").expect(200);
+    test("GET 200: should respond with an array of topics object that have slug and description properties and status 200", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const topics = response.body;
+          expect(topics).toBeInstanceOf(Array);
```

Comment:
We need to check the length of the array too as this will make sure we get the expected number of topics back as well as protect against a false positive test (from the forEach not being entered) in the event of no array being returned at all. You could do this instead of line 23. 😄